docs/sources/variables/filter-variables-with-regex.md

```diff
@@ -0,0 +1,131 @@
+# Filter variables with regex
+
+Using the Regex Query option, you filter the list of options returned by the variable query or modify the options returned.
+
+This page shows how to use regex to filter/modify values in the variable dropdown
+
+Using the Regex Query Option, you filter the list of options returned by the Variable query or modify the options returned.
```

Comment:
A link to JavaScript regex or something could be useful I think. Not sure if https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions is a good source. Maybe @torkelo have any input on this?

Reply:
I added this link. Please let me know if we need a different one.